config/environments/test.rb

```diff
@@ -28,9 +28,6 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
 
-  # Raise exceptions instead of rendering exception templates
-  config.action_dispatch.show_exceptions = false
-
```

Comment:
I changed it, as my controller was supposed to throw `RecordNotFound` in some cases. I wanted to test this particular case, but then I was getting status 500 instead of expected 404. 

I don't think this option affects a test console output itself. Probably you can see it only if you debug tests and inspect the page content (which is not very handy in console anyway) or if you use Selenium tests and a real web browser (which we don't use in LARA). I feel this option is mostly useful in dev mode, however I can try to look for another solution if you find this option useful in tests too.